src/ui/server/graphql/queries/application.ts

```diff
@@ -38,6 +38,11 @@ const applicationQuery = gql`
           }
           validExporterLocation
         }
+        nominatedLossPayee {
+          isAppointing
+          locatedInUk
+          name
+        }
```

Comment:
do we need to also query the UK and international tables?

Reply:
Eventually yes, but we don't need any of the UI GQL queries right now. Just put this in as a place holder as it'll be the first part of the whole section that will need to be queried.

Also need to think about how the sensitive financial data will be returned after encryption.